package/src/target.rs

```diff
@@ -62,13 +62,26 @@ pub enum RackTopology {
     SingleSled,
 }
 
+/// Topology of the ClickHouse installation within the rack.
+#[derive(Clone, Debug, strum::EnumString, strum::Display, ValueEnum)]
+#[strum(serialize_all = "kebab-case")]
+#[clap(rename_all = "kebab-case")]
+pub enum ClickhouseTopology {
```

Comment:
I don't believe this is strictly needed. Why do we need to build differently depending upon the way we *run* clickhouse. I think you can always just build `Clickhouse`, `ClickhouseServer` and `ClickhouseKeeper` zones and package them up, but only deploy zones as decided at runtime. Isn't that exactly what we plan to do for production? 

You already have to change the hardcoded variables in RSS to run multi-node, so we can just leave that for now until Reconfigurator is up and running. We'll only start regular `Clickhouse` zones when configured in the RSS Plan, and start replicated ones otherwise.

Reply:
I'm not sure I follow here. This is just about building packages, not about what gets run via RSS, right? If so, why can't we build all the zone packages without specifying here? Then if we chose to remove all single-node functionality, we just remove that zone type. We don't also have to remove this option.

What am I missing?